### PR TITLE
Additional pointwise ops

### PR DESCRIPTION
**Pull request by crutcher**
Unify `TestDTensorOpsTest` methods.
Re-gen lagging ops db.

### CODE REVIEW

File: spmd/tensor/ops/pointwise_ops.py [hunk 1]
```diff
@@ -28,10 +28,20 @@
 pointwise_ops = [
     # please keep the entries below alphabetically sorted
     "aten.abs.default",
+    "aten.acos.Scalar",
```
Comment (wanchaol) [inline spmd/tensor/ops/pointwise_ops.py:31]: I think these op overloads are not actually going through `__torch_dispatch__`, so even if we added there they might not really being used, they are getting preprocessed and passed to a op that goes through dispatch op (i.e. aten.cos.default). For ops that are actually going through, they are mainly the ops in native_functions.yaml and have a `dispatch` bounded method https://github.com/pytorch/pytorch/blob/e217b30b0fc98df226654cef0617eb41a177531f/aten/src/ATen/native/native_functions.yaml#L474
`

File: spmd/tensor/ops/pointwise_ops.py [hunk 2]
```diff
@@ -259,6 +333,8 @@
     "aten.nextafter.default",
     "aten.nextafter.out",
     "aten.nextafter_.default",
+    "aten.ones_like.default",
```
Comment (wanchaol) [inline spmd/tensor/ops/pointwise_ops.py:336]: we already have ones_like in https://github.com/pytorch/tau/blob/eba6bdf1ec50cfa2d86d032b66da1a356b0eeefc/spmd/tensor/ops/tensor_ops.py#L94 which is not really a pointwise rule

File: spmd/tensor/ops/pointwise_ops.py [hunk 3]
```diff
@@ -346,6 +436,8 @@
     "aten.xlogy_.OutScalar_Other",
     "aten.xlogy_.Scalar_other",
     "aten.xlogy_.Tensor",
+    "aten.zeros_like.default",
```
Comment (wanchaol) [inline spmd/tensor/ops/pointwise_ops.py:439]: similarly, zero_like already in other rules

File: spmd/testing/dtensor_lagging_op_db.py [hunk 4]
```diff
@@ -312,6 +293,25 @@
     ("lu_unpack", ""),
     ("mH", ""),
     ("mT", ""),
+    ("masked.amax", ""),
```
Comment (wanchaol) [inline spmd/testing/dtensor_lagging_op_db.py:296]: what's the pointwise op that is really enabled in this PR? it looks like we just re-generate the op db with additional entries of operator tests, but nothing get removed from this `xfail` list.

Comment (crutcher) [reply spmd/testing/dtensor_lagging_op_db.py:296]: A lot of things got removed from the xfail list.

Comment (crutcher) [reply spmd/testing/dtensor_lagging_op_db.py:296]: All of these were removed from xfail. I was given the mandate of "as many as I could"; and this was me pulling on that and related strings.

And I asked you and @aazzolini a fair number of questions.

If you don't want me contributing, ok. 

```
    xfail("clamp"),
    xfail("clamp_max"),
    xfail("clamp_min"),
    xfail("fmax"),
    xfail("fmin"),
```

**Pull request closed without merging**